src/pages/ResourceForm.tsx

```diff
@@ -27,13 +30,15 @@ const StyledPanel = styled.div`
   flex-direction: column;
   width: 100%;
   min-height: 10rem;
-  padding: 2rem 2rem 1rem;
+  padding-top: 2rem;
+  padding-bottom: 1rem;
 `;
 
 const StyledButtonWrapper = styled.div`
   display: flex;
-  justify-content: space-between;
   width: 100%;
+  justify-content: space-between;
+  max-width: ${({ theme }) => theme.breakpoints.values.md + 'px'};
```

Comment:
Bør man ikke unngå bruk av px?

Reply:
jo, men dette er terskelverdier for ulike devicer, globalt definert.  (dessuten stjålet fra NVA :-)